app/src/androidTest/java/com/android/sample/ui/activity/CreateActivityScreenTest.kt

```diff
@@ -1,99 +1,197 @@
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].surname,
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].age.toString(),
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Button(
+            onClick = { /* no-op */},
+            modifier = Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
+        ) {
+          Icon(
+              Icons.Filled.PersonRemove,
+              contentDescription = "remove attendee",
+          )
         }
-        composeTestRule.onNodeWithTag("inputPriceCreate").assertExists()
-        composeTestRule.onNodeWithTag("inputPriceCreate").assertIsDisplayed()
+      }
     }
 
-    @Test
-    fun createActivityScreen_displaysPlacesLeftField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
-        }
-        composeTestRule.onNodeWithTag("inputPlacesCreate").assertExists()
-        composeTestRule.onNodeWithTag("inputPlacesCreate").assertIsDisplayed()
+    composeTestRule.onNodeWithText("John").assertExists()
+    composeTestRule.onNodeWithText("Doe").assertExists()
+    composeTestRule.onNodeWithText("30").assertExists()
+  }
 
-    }
+  @Test
+  fun removeButtonRemovesAttendee() {
+    var attendees = listOf(SimpleUser("John", "Doe", 30))
 
-    @Test
-    fun createActivityScreen_displaysLocationField() {
-        composeTestRule.setContent {
-            CreateActivityScreen(mockViewModel, mockNavigationActions)
+    composeTestRule.setContent {
+      Row(
+          modifier = Modifier.padding(8.dp).background(Color(0xFFFFFFFF)),
+      ) {
+        Text(
+            text = attendees[0].name,
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].surname,
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Spacer(modifier = Modifier.width(8.dp))
+        Text(
+            text = attendees[0].age.toString(),
+            modifier = Modifier.padding(8.dp),
+            style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
+        )
+        Button(
+            onClick = { attendees = attendees.filter { it != attendees[0] } },
+            modifier = Modifier.width(40.dp).height(40.dp).testTag("removeAttendeeButton"),
+        ) {
+          Icon(
+              Icons.Filled.PersonRemove,
+              contentDescription = "remove attendee",
+          )
```

Comment:
Again, I don't understand why the code is written this way. Where do you call the code to test ?